### ISSUE TITLE
Killing 3main fails to restart nvOC when zhash ewbf is mining

### ISSUE THREAD

**Issue by LuKePicci** [papampi/nvOC_by_fullzero#160]
I just noticed something strange before starting my tests for my last PR. Killing 3main usually restart the whole nvOC logic successfully (I tested and still works with ethminer),
This is not the case with new ewbf for Zhash, 3main restarts correctly, but it gets stuck somewhere and does not kill the currently running ewbf instance. I think WTM_SWITCHING killing of 3main works anyway. Seems just an issue with 3main getting stuck.

Did somebody else notice such behavior?

**Comment by papampi** [comment]
I just tested 2 times on latest commit, killing 3main kills miner and start it again

```
Thu Jul 26 02:59:06 +0430 2018 - Miner is running, 3main killing miner
  SCREEN -c /home/m1/NVOC/mining/screenrc-miner -dmSL miner /home/m1/NVOC/mining/miners/z_ewbf/latest/miner --fee 0 --server us-east.equihash-hub.miningpoolhub.com --user papampi.nv100 --pass x --port 20595 --algo 144_5 --pers BgoldPoW
```

**Comment by LuKePicci** [comment]
Just tested again, twice, Actually it works but it takes an huge time between the `pkill -f 3main` command and the miner kill. I measured: 3 minutes and 20 seconds … I think there is some nvidia command which is gettings stucked for this huge time, I will check if it is feasible to run it in background to avoid locking. It is something related to nvidia-driver probably.

**Comment by papampi** [comment]
On my test rig it restarts like others under 10-15 seconds

**Comment by LuKePicci** [comment]
It stucks on GPULogoBrightness setting
![image](https://user-images.githubusercontent.com/8722358/43232633-474bad78-9072-11e8-96b2-11a5f7a88541.png)

**Comment by LuKePicci** [comment]
I moved the logo setting at the end, together within the final call to nvidia-settings. The problem is not the led setting itself but the call to nvidia-settings which is taking very long while z_ewbf is running. Should be fixed in my branch.

**Comment by papampi** [comment]
I have not set GPU brightness, so that could be the reason its working fine.

**Comment by LuKePicci** [comment]
Fixed in #159
